cool_seq_tool/sources/uta_database.py

```diff
@@ -890,18 +890,20 @@ async def get_transcripts_from_gene(
         if not gene and not alt_ac:
             return pd.DataFrame([], columns=columns)
 
-        if use_tx_pos:
-            pos_cond = f"""
-                AND {start_pos} + T.cds_start_i
-                    BETWEEN ALIGN.tx_start_i AND ALIGN.tx_end_i
-                AND {end_pos} + T.cds_start_i
-                    BETWEEN ALIGN.tx_start_i AND ALIGN.tx_end_i
-                """
-        else:
-            pos_cond = f"""
-                AND {start_pos} BETWEEN ALIGN.alt_start_i AND ALIGN.alt_end_i
-                AND {end_pos} BETWEEN ALIGN.alt_start_i AND ALIGN.alt_end_i
-                """
+        pos_cond = ""
+        if start_pos and end_pos:
```

Comment:
It would probably be safer to check whether each of these values `is None`, since `0` also counts as falsey

Reply:
DANGIT, yeah you're right. I need a nap

Reply:
I think I should likely add a test case for this as well